code/game/objects/items/RCD.dm

```diff
@@ -843,7 +843,8 @@ RLD
 		return FALSE
 
 	if(checkResource(machinery_data["cost"][blueprint], user) && blueprint)
-		if(do_after(user, machinery_data["delay"][blueprint], target = A))
+		//"cost" is also used as delay here, because honestly everyone always made cost and delay the same and its a lot easier to handle this way
```

Comment:
```suggestion
		//"cost" is relative to delay at a rate of 10 matter/second  (1matter/decisecond) rather than playing with 2 different variables since everyone set it to this rate anyways.
```